app/controllers/tournaments_controller.rb

```diff
@@ -20,6 +40,13 @@ def scores
   end
 
   def success
+    begin
+      results = Result.select("results.answers").where(tournament_id: params[:id])
```

Comment:
Avoid using raw SQL, I think this might work instead:
```suggestion
      results = Result.select(:answers).where(tournament_id: params[:id])
```